api/qr.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Photobooth\Utility\QrCodeUtility;
+
+class QrCodeClass
+{
+    public function getWifiQrCode($qrConfig)
+    {
+        // hidden
+        if ($qrConfig['wifi_ssid_hidden'] == true) {
+            $hidden = (string) ',H:true';
+        } else {
+            $hidden = (string) '';
+        }
+
+        // secure
+        if ($qrConfig['wifi_secure'] == 'WPA' || $qrConfig['wifi_secure'] == 'WPA2') {
+            $sec = (string) 'T:WPA;';
+        } elseif ($qrConfig['wifi_secure'] == 'WEP') {
+            $sec = (string) 'T:WEP;';
+        } else {
+            $sec = (string) '';
+        }
+        $wifi = 'WIFI:S:' . $qrConfig['wifi_ssid'] . ';' . $sec . 'P:' . $qrConfig['wifi_pass'] . $hidden . ';';
+
+        $result = QrCodeUtility::create($wifi);
+        header('Content-Type: ' . $result->getMimeType());
```

Comment:
when moved to QrCodeUtility only return the QRCode, keep responses for the API endpoints—that way the codes can be utilized and generated for multiple purposes.